tests/4.5/target_teams_distribute_parallel_for/test_target_teams_distribute_parallel_for_dist_schedule.c

```diff
@@ -0,0 +1,62 @@
+//===-test_target_teams_distribute_parallel_for_dist_schedule.c-===//
+//
+// OpenMP API Version 4.5 Nov 2015
+//
+// testing the combined construct target teams distribute parallel for
+// This test will check if dist_schedule(static, 4) taking effect
+//
+//===---------------------------------------------------------------------===//
+
+#include <omp.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "ompvv.h"
+
+#define THRD 10
+
+int Dist_ScheduleStatic() {
+  int ErrCount = 0;
+  int Arr[32];
+  for (int i = 0; i < 32; ++i) {
+    Arr[i] = i;
+  }
+
+  int TrackUsedThread[THRD];
+  for (int i = 0; i < THRD; ++i) {
+    TrackUsedThread[i] = -1;
+  }
+
+#pragma omp target teams distribute parallel for num_teams(2)\
+        thread_limit(THRD)  map(tofrom: TrackUsedThread[0:THRD])\
+        dist_schedule(static, 4)
+  for (int i = 0; i < 32; ++i) {
+    if (Arr[i] != i) {
```

Comment:
What is the purpose of Arr in this test?. It doesn't seem like it is doing something with the array

Reply:
Removed Arr[32]  and associated access...